src/components/AIETReporting.vue

```diff
@@ -87,10 +87,10 @@ export default {
   methods: {
     async authenticate() {
       this.isLoading = true;
-      this.isUserValid = await userAPI.verifyAIETStudent(
-        this.$refs.phoneNumberEntry.phoneNumber,
-        this.$route.params.dateOfBirth
-      );
+      this.isUserValid = await userAPI.verifyAIETStudent({
+        phone_number: this.$refs.phoneNumberEntry.phoneNumber,
+        date_of_birth: this.$route.params.dateOfBirth,
```

Comment:
is this valid -- at one point, you are sending 5 things to `UserAPI.verifyAIETStudent`; here you are sending only two things

Reply:
line 234 in aietprofileform.vue

Reply:
it's valid because the arguments are not sent individually, its an object. So the backend's job is to deconstruct the object.